client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,33 @@
+import Grid from '@material-ui/core/Grid';
+import Stack from '@mui/material/Stack';
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { Link } from 'react-router-dom';
+
+const Navbar = (): JSX.Element => {
```

Comment:
The Navbar components is part of a separate ticket - Dashboard Skeleton. Can you please include it as a part of another PR? :)

Reply:
Hi Rajiv, I was not aware of this. When I read the first ticket, it was written to build signup and login pages based on the mockup project. That's why I built the navbar too.